src/components/tree/fileTree.tsx

```
@@ -0,0 +1,132 @@
+        </div>
+      </Tree.Item>
+    );
+  },
+);
+
+interface TreeNodeProps extends Omit<TreeNode, "children"> {
+  onChange$: QRL<(event: Event, element: HTMLInputElement) => void>;
+  isChecked: boolean;
+}
+
+const TreeNode = component$<TreeNodeProps>(
```

Biome (1.9.4)

[error] Shouldn't redeclare 'TreeNode'. Consider to delete it or rename it.

'TreeNode' is defined here:


(lint/suspicious/noRedeclare)